HISTORY.rst

```diff
@@ -4,7 +4,15 @@ History
 
 0.12.4 (unreleased)
 -------------------
-* In tests, set xclim' missing value option to ``skip``. As of xclim 0.45, missing value checks are applied to ``fit`` indicator, meaning that parameters will be set to None if missing values are found in the fitted time series. Wrap calls to ``fit`` with ``xclim.set_options(check_missing="skip")`` to reproduce the previous behavior of xclim.
+
+Breaking changes
+^^^^^^^^^^^^^^^^
+* In tests, set `xclim`'s missing value option to ``skip``. As of `xclim` v0.45, missing value checks are applied to the ``fit`` indicator, meaning that parameters will be set to `None` if missing values are found in the fitted time series. Wrap calls to ``fit`` with ``xclim.set_options(check_missing="skip")`` to reproduce the previous behavior of xclim.
+* `RavenPy` processes and tests that depend on remote THREDDS/GeoServer now allow for optional server URL and file location targets. These can be set with the following environment variables:
+    * `RAVENPY_THREDDS_URL`: URL to the THREDDS-hosted climate data service. Defaults to `https://pavics.ouranos.ca/twitcher/ows/proxy/thredds`.
```

Comment:
This new `RAVENPY_THREDDS_URL` is brand new, why would it be breaking if there is nothing existing to break?

For the new `RAVENPY_GEOSERVER_URL`, since the old `GEO_URL` still supported, should normally not listed as a breaking change.

Reply:
I saw the updated wording but I am not sure you understood my point.  My point was why `RAVENPY_THREDDS_URL` and `RAVENPY_GEOSERVER_URL` are listed in the "breaking change" section since they do not break any existing behavior.

For me "breaking change" means change of behavior that is not compatible with the previous behavior.  Not that "breaking change" means important changes that need to be highlighted.  Which definition of "breaking change" is in effect here?